jni/iri_pearldiver_exlib.c

```diff
@@ -13,11 +13,33 @@
 #include "../src/trinary.h"
 
 JNIEXPORT jboolean JNICALL
-Java_com_iota_iri_crypto_PearlDiver_exlibInit(JNIEnv *env, jclass clazz)
+Java_com_iota_iri_crypto_PearlDiver_exlibInit(JNIEnv *env,
+                                              jclass clazz,
+                                              jbyteArray broker_host,
+                                              jint broker_host_len)
 {
-    if (!dcurl_init())
-        return JNI_FALSE;
-    return JNI_TRUE;
+    jboolean ret = JNI_TRUE;
+
+    /*********** Get the Byte array from Java byte Array *************/
```

Comment:
Avoid using comment redundant comment style. Use plain `/* The content */` instead.

Reply:
I also refactor the other comments in `jni/iri_peraldiver_exlib.c`.